src/detections/chart/by-mode.js

```diff
@@ -0,0 +1,7 @@
+import { groupBy } from 'lodash'
+
+import { countByMode } from 'detections/chart/count-by-mode'
+import { byClass } from 'detections/chart/by-class'
+
+export const byMode = (items, mode) => Object.entries(groupBy(items, byClass))
```

Comment:
Let's try to emulate the following folder structure:
![image](https://user-images.githubusercontent.com/23569938/129048514-78af1c8e-52cc-433b-a84a-4e58f95948b2.png)

That way, we can create an `index.js` file which would look like this:
![image](https://user-images.githubusercontent.com/23569938/129048569-5c82b9b8-287a-47e4-aedf-4874cee2959a.png)

Reply:
done